intranet/static/js/dashboard/announcements.js

```diff
@@ -62,4 +69,4 @@ $(document).ready(function() {
         var btn = $(".announcement-toggle", $(this).parent());
         announcementToggle.call(btn);
     });
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2758448633/issues/source?bid=3176160&fileBranchId=3355598#l72)